src/transformers/modeling_utils.py

```diff
@@ -1764,7 +1767,7 @@ def floating_point_ops(
         return 6 * self.estimate_tokens(input_dict) * self.num_parameters(exclude_embeddings=exclude_embeddings)
 
 
-class PreTrainedModel(nn.Module, ModuleUtilsMixin, PushToHubMixin, PeftAdapterMixin):
+class PreTrainedModel(nn.Module, ModuleUtilsMixin, ContinuousMixin, PushToHubMixin, PeftAdapterMixin):
```

Comment:
can we add it instead as a mixin to `GenerationMixin`? 👀 

rationale: many `PreTrainedModel`-derived instances can't generate, all `GenerationMixin`-derived instances can. That way, we spare those models of any additional requirement, present and future

Reply:
okay does not sound bad ye[